domain/domain.go

```diff
@@ -55,7 +55,7 @@ func (do *Domain) loadInfoSchema(txn kv.Transaction) (err error) {
 	if info != nil && schemaMetaVersion <= info.SchemaMetaVersion() {
 		// info may be changed by other txn, so here its version may be bigger than schema version,
 		// so we don't need to reload.
-		log.Debugf("schema version is still %d, no need reload", schemaMetaVersion)
+		log.Debugf("[ddl] schema version is still %d, no need reload", schemaMetaVersion)
```

Comment:
We set the tag base on the message it logs, so multiple package may use the same tag.